react/src/components/atoms/forms/InputTextFuzzy/index.js

```diff
@@ -69,13 +123,19 @@ class InputTextFuzzy extends React.Component {
         value: this.state.value,
         disabled: this.props.disabled,
         id: this.props.inputId,
-        onKeyDown: (event, { newHighlightedSectionIndex, newHighlightedItemIndex }) => {
+        onFocus: this.handleFocus,
+        onBlur: (event) => {
+          event.persist();
```

Comment:
don't think we need to persist this event unless we want to pass a custom callback